exporter/redis.go

```diff
@@ -859,7 +859,7 @@ func (e *Exporter) scrapeRedisHost(scrapes chan<- scrapeResult, addr string, idx
 		if err != nil {
 			switch err {
 			case errNotFound:
-				log.Debugf("Key %v not found when trying to get type and size.", err)
+				log.Debugf("Key '%v' not found when trying to get type and size.", k.key)
```

Comment:
I think you can make this `%s` as k.key is a string.

Reply:
You are right.